test/integration/events_create_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class EventsCreateTest < ActionDispatch::IntegrationTest
+  setup do
+    Event.delete_all
+  end
+
+  test 'valid event information saves' do
+    assert_difference 'Event.count', 1 do
+      create :event, :updated_today
+    end
+  end
+
+  test 'find_or_initialize_by finds duplicate event by source_id and source' do
```

Comment:
We can remove this test, as it is exclusively testing a Rails method, `find_or_initialize_by`.